java/impl/aws/src/main/java/co/worklytics/psoxy/AwsApiDataModeHybridHandler.java

```diff
@@ -0,0 +1,133 @@
+package co.worklytics.psoxy;
+
+import co.worklytics.psoxy.aws.AwsContainer;
+import co.worklytics.psoxy.aws.DaggerAwsContainer;
+import co.worklytics.psoxy.aws.request.APIGatewayV1ProxyEventRequestAdapter;
+import co.worklytics.psoxy.aws.request.APIGatewayV2HTTPEventRequestAdapter;
+import co.worklytics.psoxy.aws.request.LambdaEventUtils;
+import co.worklytics.psoxy.gateway.HttpEventRequest;
+import co.worklytics.psoxy.gateway.HttpEventResponse;
+import co.worklytics.psoxy.gateway.impl.ApiDataRequestHandler;
+import com.amazonaws.services.lambda.runtime.Context;
+import com.amazonaws.services.lambda.runtime.RequestStreamHandler;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.extern.java.Log;
+import org.apache.commons.lang3.exception.ExceptionUtils;
+import org.apache.commons.lang3.tuple.Pair;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+
+/**
+ * AWS lambda entrypoint that can handle API data requests via any of:
+ *   - cloud function URL invocations
+ *   - API Gateway events
+ *   - SQS events ( asynchronous processing of API data requests, that originally came from one of the above)
+ */
+@Log
+public class AwsApiDataModeHybridHandler implements RequestStreamHandler {
```

Comment:
new hybrid handler that will be entry point for API data connections, if they support async processing